.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.7.2
```

Comment:
To jest config do `asdf`. 
Powinno wystarczyć wpisać `asdf install` jeśli macie go skonfigoruwanego i w teorii dla tego projektu powinna być ustawiona wersja pythona 3.7.2 (w teorii, bo sam tego jeszcze nie testowałem)